src/30_Advanced/Embedding_videos_in_amp-carousel.html

```diff
@@ -0,0 +1,140 @@
+<!--
```

Comment:
We should rename this sample to "Video Carousels with amp-carousel" corresponding to the already existing "Image Galleries with amp-carousel" sample.